src/Raven.Server/Documents/Handlers/DocumentHandler.cs

```diff
@@ -40,6 +41,41 @@ public Task Head()
             }
         }
 
+        [RavenAction("/databases/*/docs/$", "GET")]
```

Comment:
We assumed that we want to have an endpoint to show a single document (not in array). I will remove it tomorrow.